packages/now-node/src/index.ts

```diff
@@ -41,12 +46,39 @@ async function downloadInstallAndBundle({
   return { entrypointPath, entrypointFsDirname };
 }
 
-async function compile(entrypointPath: string, entrypoint: string, config: CompilerConfig): Promise<Files> {
+async function compile(
+  entrypointPath: string,
+  entrypoint: string,
+  config: CompilerConfig,
+  workPath: string,
+  { isDev, filesChanged = [], filesRemoved = [] }: Meta
+): Promise<{ preparedFiles: Files, watch: string[] }> {
   const input = entrypointPath;
   const inputDir = dirname(input);
   const rootIncludeFiles = inputDir.split(sep).pop() || '';
-  const ncc = require('@zeit/ncc');
-  const { code, map, assets } = await ncc(input, { sourceMap: true, sourceMapRegister: true });
+  const nccOptions = { sourceMap: true, sourceMapRegister: true };
```

Comment:
Potentially a nice convenience if we could also pass nccOptions through CompilerConfig:

```suggestion
  const nccOptions = { ...config.nccOptions, sourceMap: true, sourceMapRegister: true };
```